js/google_ima.js

```diff
@@ -1248,6 +1262,8 @@ require("../html5-common/js/utils/utils.js");
                 this.setVolume(this.savedVolume);
                 this.savedVolume = -1;
               }
+            } else {
```

Comment:
style issue. can you space out the curly braces so it matches the rest of the file.